src/askui/tools/agent_os.py

```diff
@@ -302,3 +302,16 @@ def set_display(self, displayNumber: int = 1) -> None:
                 Defaults to `1`.
         """
         raise NotImplementedError
+
+    @abstractmethod
+    def run_command(self, command: str, timeout_in_milliseconds: int = 30000) -> None:
```

Comment:
I think this is too verbose:

```suggestion
    def run_command(self, command: str, timeout: int = 30000) -> None:
```

or

```suggestion
    def run_command(self, command: str, timeout_ms: int = 30000) -> None:
```

or

```suggestion
    def run_command(self, command: str, timeout_in_ms: int = 30000) -> None:
```